include/ddc/detail/tagged_vector.hpp

```diff
@@ -232,6 +232,7 @@ template <class ElementType, class Tag>
 class TaggedVectorConversionOperators<TaggedVector<ElementType, Tag>>
 {
 public:
+    // NOLINTBEGIN(google-explicit-constructor)
```

Comment:
Could you please comment why you need this?
This seems to be implicit conversion?

Reply:
The check also work cast operators, see https://clang.llvm.org/extra/clang-tidy/checks/google/explicit-constructor.html

Reply:
Sure